packages/ui/src/components/va-divider/VaDivider.vue

```diff
@@ -32,6 +32,7 @@ export default defineComponent({
       default: 'center',
       validator: (value: string) => ['left', 'right', 'center'].includes(value),
     },
+    color: { type: String, default: 'backgroundBorder' },
```

Comment:
The `default` value won't apply, because it is just a string. You need to handle this with `useColors` hook and after that it will be converted to the CSS color. [Example](https://github.com/epicmaxco/vuestic-ui/blob/develop/packages/ui/src/components/va-avatar/VaAvatar.vue#L70-L78), and how it should be [applied](https://github.com/epicmaxco/vuestic-ui/blob/develop/packages/ui/src/components/va-avatar/VaAvatar.vue#L132) in the `style` tag.

Reply:
Because of it we have a regression, here is difference:
<details>
<summary>production</summary>

<img width="1098" alt="image" src="https://user-images.githubusercontent.com/29167241/226403319-88ab0910-b16a-4916-a884-bfe4b3c463bd.png">
</details>
and
<details>
<summary>your implementation</summary>

<img width="975" alt="image" src="https://user-images.githubusercontent.com/29167241/226403248-088c82c1-796d-49c1-82a9-2558f269fb46.png">
</details>

Reply:
Thanks!

Reply:
@rustem-nasyrov  Should I use the same hook from VaAvatar.vue by importing it or make a new one?

Reply:
@Yasir761, you need to import it and use it like it done in `VaAvatar` component.

Reply:
I fixed it to not bother @Yasir761 with it. 👌